style.css

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
- [ ] [OPTIONAL] In order to have a better experience with CSS, I recommend starting all your CSS files with some custom resets. You can do it by adding some rules at the beginning of the file like `* { margin: 0; }` or similar. You can learn more [here](https://www.joshwcomeau.com/css/custom-css-reset/) and, obviously, you are free to customize it as much as you want.